keep/api/models/db/migrations/versions/2024-08-26-17-01_1aacee84447e.py

```diff
@@ -0,0 +1,29 @@
+"""Store timeunit for Rule for better UX
+
+Revision ID: 1aacee84447e
+Revises: 1c650a429672
+Create Date: 2024-08-26 17:01:21.263004
+
+"""
+
+import sqlalchemy as sa
+import sqlmodel
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "1aacee84447e"
+down_revision = "1c650a429672"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    with op.batch_alter_table("rule", schema=None) as batch_op:
+        batch_op.add_column(
+            sa.Column("timeunit", sqlmodel.sql.sqltypes.AutoString(), nullable=False, server_default="seconds")
```

Comment:
Wouldn't it be easier just to store it as seconds and always translate it on the front end? (e.g. if it's divided by 3600 - show errors if its divided by 84400, show days?)

Reply:
What if I want to avoid seeing minutes and like to see 300 seconds - i'd like to think that I've entered them like this for a reason